### ISSUE TITLE
preselect notebook from query-widget where possible

### ISSUE THREAD

**Issue by andydrop** [orgzly/orgzly-android#154]
When the "add note" button of the query widget is pressed, and the shown query specifys one or more notebooks. the first notebook in the query should be preselected as the notebook the new note goes to, by default.

Would be a great improvement for my "b.shoppinglist .i.done". When i try to add a note, it now always comes up with main.org as the preselected notebook (maybe it was the first one i synced)
I would like it to open with the shoppinglist preselected, so the new note goes right there.

**Comment by nevenz** [comment]
You can set which notebook to use in Settings (Notebooks -> Default notebook). It's used when sharing to Orgzly and from widget.

Since search can display notes from multiple notebooks, I think it would be too confusing to use the first one (first search result doesn't mean that much)

But we could use a notebook if it's part of the query, like in your example. If there's `b` in there, use that notebook when `+` is pressed. If not, use default from Settings.

(BTW, first notebook in the list is now used, if you don't have notebook specified in "Default notebook". Which one is that depends on configured ordering. Either by name or last-modified.)

**Comment by andydrop** [comment]
Yeah, I agree on not depending the notebook on the result list. That could be too confusing. 

I just meant to depend it on the query, if that contains a b. If it contains more than one b. let it depend on the first used. By ordering the b. in the query, the user can select the default book for adding 

_Sent from my Fairphone FP2 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by somini** [comment]
Can't you just add a field on the Search Query to select the "Default Notebook for Addition", a per-query setting, instead of guessing from the query? You can guess it once to set the default.

**Comment by nevenz** [comment]
I do think an explicit configuration would be better, yes. But I don't know if it should be part of a query, perhaps it can be a per-widget setting (like the query itself is). Unless there are users who often change the query for a widget, as opposed to setting it once and having multiple widgets if needed.
